README.markdown

```diff
@@ -902,6 +902,58 @@ if let number1 = number1 {
 
 Guard statements are required to exit in some way. Generally, this should be simple one line statement such as `return`, `throw`, `break`, `continue`, and `fatalError()`. Large code blocks should be avoided. If cleanup code is required for multiple exit points, consider using a `defer` block to avoid cleanup code duplication.
 
+### Switch Statements
+
+When at all possible, avoid using the `default` clause in `switch` statements, instead prefer to explicitly enumerate the unexpected conditions. This is especially true with `enum` data types. The rationale being if a new `case` is added to an `enum`, we want the compiler to generate an error informing us of the unhandled case. If we used a `default` clause, the new case would fall through to that clause and that default action may not be how that case should be handled.
+
+**Preferred:**
+```swift
+switch someDirection {
```

Comment:
The formatting of these are wrong. This is how XCode did this (and how we usually do it):
```
        switch someDirection {
        case west:
            print("Go west, young man!")
        case north,
             south,
             east:
            print("Go somehere else!")
        }
```

Same for all the examples below.